### PR TITLE
Add trace to Redux Devtools Extension

### PR DESCRIPTION
**Pull request by codalife**
https://github.com/zalmoxisus/redux-devtools-extension/blob/master/docs/Features/Trace.md

### CODE REVIEW

**Review by KevinGrandon — CHANGES_REQUESTED** [review]
Thanks for the PR! Looks good to me, but we should remove the package-lock.json file since we use yarn for fusion development. Happy to send a contribution to your branch if helpful.

**Comment by codalife** [comment]
> Thanks for the PR! Looks good to me, but we should remove the package-lock.json file since we use yarn for fusion development. Happy to send a contribution to your branch if helpful.

Removed lock. Rookie mistake

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "fusion-plugin-react-redux",
   "description": "Integrates React and Redux to a Fusion.js application",
-  "version": "1.1.2",
+  "version": "1.1.3",
```
Comment (chrisdothtml) [inline package.json]: You can also remove this change as we will bump the version through our release process

**Comment by mlmorg** [comment]
We still want the `package.json`, you just don't need to bump the version since it will be handled by our release process.

**Review by KevinGrandon — DISMISSED** [review]

**Review by chrisdothtml — APPROVED** [review]

**Comment by KevinGrandon** [comment]
!merge

**Pull request merged**